cropduster/tests/test_gifsicle.py

```diff
@@ -0,0 +1,44 @@
+from io import open, BytesIO
+import os
+import shutil
+import tempfile
+
+from PIL import Image, ImageSequence
+from io import BytesIO
+
+from django import test
+from django.core.files.storage import default_storage
+from django.conf import settings
+
+from .helpers import CropdusterTestCaseMediaMixin
+
+
+class TestUtilsImage(CropdusterTestCaseMediaMixin, test.TestCase):
```

Comment:
For tests where Pillow is < 5.4, perhaps the gif tests should raise `unittest.SkipTest`; there could also be a test that skips if Pillow >= 5.4 and does a `self.assertRaises()`